NEWS.md

```diff
@@ -1,6 +1,19 @@
 
 # rlang 0.3.4.9000
 
+* `call2()` now uses `pairlist2()` internally. This brings its
+  performance closer to that of `base::call()`. This also means it now
```

Comment:
I think this oversells it a bit given your benchmarks - maybe just say "considerably improves performance"?